lib/summarize.js

```diff
@@ -48,6 +48,11 @@ module.exports = function () {
       dup.failed = true
     }
 
+    if (res.plans.length < 1) {
+      dup.failed = true
+      process.exit(1)
```

Comment:
Maybe we could emit an error rather than exit directly?
It seems both `res.fail.length > 0` and `res.plans.length < 1` mean a failure, but we handle them differently here.

BTW,  could you give an example to show the problem this pr is going to resolve?